### PR TITLE
AIX 11.0.16 requires the 16.1 XL C++ Runtime

### PR DESCRIPTION
**Pull request by pshipton**
Fixes https://github.com/eclipse-openj9/openj9-docs/issues/932

### CODE REVIEW

**Comment by pshipton** [comment]
jenkins doc stage

File: docs/version0.33.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+### Binaries and supported environments
+
+OpenJ9 release 0.33.0 supports OpenJDK 8, 11, 17, and 18.
+
+To learn more about support for OpenJ9 releases, including OpenJDK levels and platform support, see [Supported environments](openj9_support.md).
+
+### ![Start of content that applies to Java 11](cr/java11.png) AIX requires the XL C++ Runtime
+
+AIX requires the [16.1 XL C++ Runtime](https://www.ibm.com/support/pages/fix-list-xl-cc-runtime-aix#161X) to accommodate a security update to the HarfBuzz text shaping library.
+
```
Comment (doveye) [inline docs/version0.33.md:43]: There's a mismatch between the link further up the topic and the title of this section, so the link doesn't work. But I think the words could use tweaking anyway to clarify that this is a change in level, to be more consistent with the text in the supported env topic, and to use a more official name for the runtime (which also clarifies that it's a third-party thing). Suggest something like:

> **Later level of XL C++ Runtime required on AIX**
> 
> AIX OpenJ9 builds now require version 16.1 of the IBM XL C++ Runtime for AIX (the previous required level was 13.1.3). This later level is required to accommodate a security update to the HarfBuzz text shaping library.

I also found this topic that mentions XL C++: https://eclipse-openj9.github.io/openj9-docs/configuring/#setting-resource-limits-aix-linux-and-macos. Looks like the Java 14 icon and text needs changing to 11.

**Review by doveye — CHANGES_REQUESTED** [review]

**Comment by pshipton** [comment]
jenkins doc stage

**Comment by pshipton** [comment]
jenkins doc stage

**Comment by pshipton** [comment]
jenkins doc stage

**Comment by pshipton** [comment]
jenkins doc stage

**Comment by pshipton** [comment]
@doveye this is ready for another review.

File: docs/version0.33.md (hunk 1 above)
Comment (doveye) [reply docs/version0.33.md:41]: Ok, I saw the change in levels in the build env table but I guess that's different.

**Review by doveye — APPROVED** [review]
LGTM

**Comment by doveye** [comment]
Had to prioritise other things yesterday. Checked again today though and the changes look good, thanks.

**Pull request merged**